contracts/tests/sbtc-deposit.test.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  alice,
+  deposit,
+  registry,
+} from "./helpers";
+import { test, expect, describe } from "vitest";
+import { txOk, filterEvents, rov } from "@clarigen/test";
+import { CoreNodeEventType, cvToValue } from "@clarigen/core";
+import { uintCV } from "../node_modules/@stacks/transactions/dist/clarity/types/intCV";
+import { principalCV } from "../node_modules/@stacks/transactions/dist/clarity/types/principalCV";
+import { tupleCV } from "@stacks/transactions";
+
+
+describe("sBTC deposit contract", () => {
+  describe("complete deposit contract setup", () => {
+    
+    test("Call complete-deposit-wrapper placeholder", () => {
+      const receipt = txOk(
+        deposit.completeDepositWrapper({
+          txid: Uint8Array.from([0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]),
```

Comment:
Nothing wrong here, just a helpful shorthand
```suggestion
          txid: new Uint8Array(32).fill(0),
```

Reply:
Updated, ty.